roles/certificates/tasks/issue.yml

```diff
@@ -17,6 +17,7 @@
         -batch
         -policy signing_policy
         -extensions signing_req
+        -passin "pass:{{ certificates_ca_password }}"
```

Comment:
Hoe future proof do you want to make this? This will show in the process list, but that may be ok if this is a throw away poc

Reply:
it is ok, but it's also not expensive to fix it, thanks! done :)